mico-core/src/main/java/io/github/ust/mico/core/resource/ServiceResource.java

```diff
@@ -382,6 +384,27 @@
             linkTo(methodOn(ServiceResource.class).getDependencyGraph(shortName, version)).withSelfRel()));
     }
 
+    /**
+     * Return yaml for a {@link MicoService} for the give shortname and version.
+     *
+     * @param shortName
```

Comment:
To be consistent with the other endpoints: Either set everything of the JavaDoc or nothing.